docs/supported_apis/dataframe_supported.rst

```diff
@@ -47,7 +53,7 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``assign``                 | `assign`_                 | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``astype``                 | `astype`_                 | Y                      |                                                    |
+| ``astype``                 | `astype`_                 | Y                      | **Omnisci**:``P``, ``int`` and ``float`` supported |
```

Comment:
I would mention that the only cast that is supported is `int <-> float`, as the note may be treated that `.astype(int)` and `.astype(float)` are certainly supported, however it's not true. Allowed type casts are ruled by [this table in omnisci's docs](https://docs.omnisci.com/v5.2.1/5_dml.html#type-cast-support) that is more convervative than pandas. For example it doesn't allow parsing strings into some other dtypes when pandas do allow:
```python
>>> pandas.Series(["1", "2", "3"]).astype(float)
0    1.0
1    2.0
2    3.0
dtype: float64
```

Reply:
Thanks, updated